java-tools/OsmAndServer/src/main/java/net/osmand/server/api/test/entity/EvalJob.java

```diff
@@ -0,0 +1,143 @@
+package net.osmand.server.api.test.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "eval_job")
+public class EvalJob {
+
+	@Id
```

Comment:
Code style: if it's not required by anything and not used, then no get/set just public fields